src/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
```

Comment:
You don't have to include this config globally for each test file, it will do in _jest.config.js_
pls read about jest configuration https://jestjs.io/docs/en/configuration and fix it